### PR TITLE
Fix test to run asynchronously and within mocha

### PR DESCRIPTION
**Pull request by kapouer**
The old tests could fail easily, see #74

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,12 +35,14 @@
   "files": [
     "lib"
   ],
-  "devDependencies": {},
+  "devDependencies": {
+    "mocha": "^5.2.0"
```
Comment (harrysarson) [inline package.json]: Mocha 6 is out now, might be worth bumping this!

Comment (miguelsolano) [reply package.json]: Will look at this tonight, before I create a new release.

Comment (miguelsolano-acs) [reply package.json]: Just update this. 👍

Comment (miguelsolano) [reply package.json]: Just upgraded this 👍

**Pull request merged**